packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -47,20 +29,11 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   // Enable per monitor DPI scaling
   SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
 
-  // Create a DispatcherQueue for this thread.  This is needed for Composition, Content, and
-  // Input APIs.
-  auto dispatcherQueueController{winrt::Microsoft::UI::Dispatching::DispatcherQueueController::CreateOnCurrentThread()};
-
-  // Create a Compositor for all Content on this thread.
-  auto compositor{winrt::Microsoft::UI::Composition::Compositor()};
-
   // Create a top-level window.
   auto window = winrt::Microsoft::UI::Windowing::AppWindow::Create();
   window.Title(windowTitle);
   window.Resize({1000, 1000});
   window.Show();
-  auto hwnd = winrt::Microsoft::UI::GetWindowFromWindowId(window.Id());
-  auto scaleFactor = ScaleFactor(hwnd);
 
   WCHAR appDirectory[MAX_PATH];
```

Comment:
I wonder if it's possible to move this elsewhere too, maybe into another helper?

Reply:
Sure I will take care of it in the following PRs.